libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -9,34 +9,48 @@ import {
 } from '@tmo/books/data-access';
 import { FormBuilder } from '@angular/forms';
 import { Book } from '@tmo/shared/models';
-import { Observable } from 'rxjs';
+import { debounceTime, distinctUntilChanged, takeUntil } from 'rxjs/operators';
+import { Subject, Observable } from 'rxjs';
 
 @Component({
   selector: 'tmo-book-search',
   templateUrl: './book-search.component.html',
-  styleUrls: ['./book-search.component.scss'],
-  changeDetection: ChangeDetectionStrategy.OnPush
```

Comment:
curious as to why you removed this?

Reply:
I added this for Task-1 as a code improvement. So removed it from Task-2 as suggested by Kandarp.
Please suggest if I can keep it.